README.adoc

```diff
@@ -68,9 +68,37 @@ include::{common-includes}/gitclone.adoc[]
 [role='command']
 include::{common-includes}/twyb-intro.adoc[]
 
-You can find your service at http://localhost:9080/artists.
+// static guide instructions:
+ifndef::cloud-hosted[]
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
+
+Now, you can access the endpoint at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists, and you can access the endpoint at `\http://localhost:9080/artists/total/<artist>` to see a particular artist’s total number of albums.
+
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+
+You can find your service at **http://localhost:9080/artists** by running the following curl command:
```

Comment:
You can find your service at  the **http://localhost:9080/artists** endpoint by running the following curl command: